backend/api/views/box.py

```diff
@@ -170,7 +170,7 @@ def upload_file(file, file_name):
     stream.write(file.read())
     stream.seek(0)
     try:
-        box_file = client.folder("0").upload_stream(
+        box_file = client.folder(str(folder_id)).upload_stream(
```

Comment:
Do you have to cast it into a str()? Isn't the parameter a string already?

Reply:
same goes to other places that use str()

Reply:
just to be safe